docs/operation/lerobot_guide.rst

```diff
@@ -0,0 +1,382 @@
+
+   Example:
+
+   .. code-block:: bash
+
+      $ python lerobot/scripts/control_robot.py record \
+      $   --robot-path lerobot/configs/robot/aloha.yaml \
+      $   --fps 30 \
+      $   --root data \
+      $   --repo-id ${HF_USER}/aloha_test \
+      $   --tags tutorial \
+      $   --warmup-time-s 5 \
+      $   --episode-time-s 30 \
+      $   --reset-time-s 30 \
+      $   --num-episodes 2
+
+.. note::
+
+   #. The :guilabel:`--num-episodes` defines the total number of episodes to be collected.
+      Therefore it will check the existing output directories for any previously recorded episodes and will start recording from the last recorded episode.
+   
+   #. The recorded data is pushed to hugging face hub by default you can set this false by using :guilabel:`--push_to_hub 0`.
+   
+
+.. note::
+
+   #. To push your dataset to Hugging Face's Hub, log in with a write-access token:
+
+      .. code-block:: bash
+
+         $ huggingface-cli login --token ${HUGGINGFACE_TOKEN} --add-to-git-credential
+
+   #. Set your Hugging Face username as a variable for ease:
+
+      .. code-block:: bash
+
+         $ HF_USER=$(huggingface-cli whoami | head -n 1)
+
+Visualizing Datasets
+====================
+
+.. video:: ../videos/visualize_dataset.mp4
+   :width: 1280
+   :height: 720
+
+
+To visualize all the episodes recorded in your dataset, run:
+
+.. code-block:: bash
+
+   $ python lerobot/scripts/visualize_dataset_html.py \
+   $   --root data \
+   $   --repo-id ${HF_USER}/aloha_test
+
+To visualize a single dataset episode from the Hugging Face Hub:
+
+.. code-block:: bash
+
+   $ python lerobot/scripts/visualize_dataset.py \
+   $   --repo-id ${HF_USER}/aloha_static_block_pickup \
+   $   --episode-index 0
+
+To visualize a single dataset episode stored locally:
+
+.. code-block:: bash
+
+   $ DATA_DIR='./my_local_data_dir' python lerobot/scripts/visualize_dataset.py \
+   $   --repo-id TrossenRoboticsCommunity/aloha_static_block_pickup \
+   $   --episode-index 0
+
+Replay Recorded Episodes
+========================
+
+Replaying episodes allows you to test the repeatability of the robot's actions.
+To replay the first episode of your recorded dataset:
+
+.. code-block:: bash
+
+   $ python lerobot/scripts/control_robot.py replay \
+   $   --robot-path lerobot/configs/robot/aloha.yaml \
+   $   --fps 30 \
+   $   --root data \
+   $   --repo-id ${HF_USER}/aloha_test \
+   $   --episode 0
+
+.. tip::
+
+   Use different :guilabel:`--fps` values to adjust the frequency of the robot actions.
+
+
+
+Training 
+========
+
+To train a policy for controlling your robot, use the following command:
+
+.. code-block:: bash
+
+   $ DATA_DIR=data python lerobot/scripts/train.py \
+   $   dataset_repo_id=${HF_USER}/aloha_test \
+   $   policy=act_aloha_real \
+   $   env=aloha_real \
+   $   hydra.run.dir=outputs/train/act_aloha_test \
+   $   hydra.job.name=act_aloha_test \
+   $   device=cuda \
+   $   wandb.enable=true
```

Comment:
Maybe keep wandb disabled for the tutorial for simplicity

Reply:
wandb is disabled as deafult